tools/ansible/README.md

```diff
@@ -54,6 +54,11 @@ To use WSU which builds WMCB for development purposes, set value of `build_wmcb`
 ```
 $ ansible-playbook -i hosts tasks/wsu/main.yaml -v -e "{build_wmcb: True}"
 ```
+
+**API rate limit exceeded error when running WSU:** \
```

Comment:
Please remove the trailing `\`

Reply:
trailing \ is to break the line

Reply:
Why is that needed in markdown?

Reply:
it's either use '\' or 2 new line for breaking the line. 
Without trailing \ the next line joins the previous line in markdown preview.

Reply:
Please make it a sub-heading under the usage section.